kornia/utils/draw.py

```diff
@@ -0,0 +1,53 @@
+from typing import Optional, Union
+
+import torch
+
+
+def rectangle(
+        image: torch.Tensor,
+        boxes: torch.Tensor,
```

Comment:
I'm checking now opencv API and they assume just two points simplifying and leaving to the user to compute top-left and bottom-right. Maybe we should consider this API. See: https://docs.opencv.org/master/d6/d6e/group__imgproc__draw.html#ga07d2f74cadcf8e305e810ce8eed13bc9

They also support passing rectangle, however we live in Python world and as for now we don't have a clear way to overload function signatures. /cc @ducha-aiki Which API should we take here ?

Reply:
@ducha-aiki Any thoughts on what to take? Opencv style is simplified, but I guess you couldn't draw a rotated rectangle unless all 4 points are specified.

Reply:
This can be a strong argument to follow the approach you propose instead of OpenCV one. The case you mention I believe that can be simply generalized by first having a `draw_line` routine that could take `BxCxHxW` and `BxNx2`. Later I believe that can be extended with `draw_rectangle` for `BxNx4` and finally `draw_polyline` `BxNxN`. BTW, been thinking about naming for the API and I believe that `K.draw_rectangle` is more explicit than just `K.rectangle`. What do you think guys ?